README.md

```diff
@@ -18,93 +20,107 @@ remotes:install_github(jonocarroll/safespace)
 
 ## Motivation
 
-Relevant blog post: https://jcarroll.com.au/2024/10/01/a-safe-space-for-learning-how-to-make-pull-requests/
+Relevant blog post:
+<https://jcarroll.com.au/2024/10/01/a-safe-space-for-learning-how-to-make-pull-requests/>
 
-If you're new to using `git` then it might be seem like making a pull request to someone
-else's project is too complicated or at risk of breaking something. This repository 
-is a _safe space_ where I encourage you to submit a pull request fixing something (
-there are many intentially borken elements here just waiting for someone to suggest 
-fixes for them.) - I will review your pull request ("PR") and provide mentoring if 
-there's anything I think could be done differently. I will then merge the pull 
-reqeust which will count towards your [Hacktoberfest 2024](https://hacktoberfest.com/participation/) tally.
+If you're new to using `git` then it might be seem like making a pull
+request to someone else's project is too complicated or at risk of
+breaking something. This repository is a *safe space* where I encourage
+you to submit a pull request fixing something ( there are many
+intentially borken elements here just waiting for someone to suggest
+fixes for them.) - I will review your pull request ("PR") and provide
+mentoring if there's anything I think could be done differently. I will
+then merge the pull reqeust which will count towards your [Hacktoberfest
+2024](https://hacktoberfest.com/participation/) tally.
 
-I only ask that you try to fix _just one thing_ in each PR; leave some fixes for 
-others, too.
+I only ask that you try to fix *just one thing* in each PR; leave some
+fixes for others, too.
 
 ## Things To Look For
 
-- typos
-- documentation (lack of, or fixes to)
-- incorrect logic
-- anything that `devtools::check()` flags
-- improvements
+-   typos
```

Comment:
Yes, all of these changes were done with Rstudio automatic something. So this raises two questions.
1. Is there a way to avoid this when forking and cloning to Rstudio? 
2. I suppose it's a good idea to check each change before committing. What's the method you use to do that in Rstudio?